packages/wonder-blocks-link/components/link-core.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {addStyle} from "wonder-blocks-core";
+import Color, {mix, fade} from "wonder-blocks-color";
+import type {SharedProps} from "./link.js";
+
+type Props = SharedProps & {
+    hovered: boolean,
+    focused: boolean,
+    pressed: boolean,
+    href: string,
+};
+
+const StyledAnchor = addStyle("a");
+export default class LinkCore extends React.Component<Props> {
+    render() {
+        const {
+            kind,
+            light,
+            hovered,
+            focused,
+            pressed,
+            children,
+            href,
+            style,
+        } = this.props;
+
+        const linkStyles = _generateStyles(kind, light);
+
+        const defaultStyles = [
+            sharedStyles,
+            !(hovered || focused || pressed) && linkStyles.default,
+            pressed
+                ? linkStyles.active
+                : (hovered || focused) && linkStyles.focus,
+        ];
+
+        return (
+            <StyledAnchor style={[defaultStyles, style]} href={href}>
+                {children}
+            </StyledAnchor>
+        );
+    }
+}
+
+const styles = {};
+
+const sharedStyles = {
```

Comment:
I remember this coming up in the ButtonCore review, as well - but was there a reason this isn't in an Aphrodite stylesheet? Given how many links we might have a on page it'd be useful to cut down on the code duplication.

Reply:
Oh, nope! Will update :)